styles/picrew/catppuccin.user.css

```diff
@@ -1039,7 +1039,7 @@
     }
     .widget_linkblock {
       border-color: @accent-color !important;
-      box-shadow: 0 0 0 0px @accent-color inset !important;
```

Comment:
> The linter was't complaining about `0px` because you can't use zero, it was saying that you don't need the `px` unit because `0rem` is the same as `0em` which is `0px`.... and so on. So you can just do `0` instead of `0px`.